app/views/admin/trends/tags/_tag.html.haml

```diff
@@ -1,6 +1,7 @@
 .batch-table__row{ class: [tag.requires_review? && 'batch-table__row--attention', !tag.requires_review? && !tag.trendable? && 'batch-table__row--muted'] }
-  %label.batch-table__row__select.batch-table__row__select--aligned.batch-checkbox
-    = f.check_box :tag_ids, { multiple: true, include_hidden: false }, tag.id
+  - unless f.nil?
```

Comment:
An alternative could be to use

```haml
- if local_assigns[:f]
```

I always found this to be quite ugly, but it would mean you could get rid of the `f: nil` argument when rendering this partial from `app/views/admin/tags/index.html.haml`.

Reply:
This seems like it could be nice, but I'm thinking I actually want to refactor this to not use that partial since I'm not sure how useful it is

Reply:
Have gone with a custom tag row partial, which emphasizes the moderation status as a whole, instead of emphasizing the trending status.